src/website/views/components/data-table/examples/portal/_complex.pug

```diff
@@ -0,0 +1,1708 @@
+            <template>
+              <div class="-text--truncate">
+                <a
+                  :id="`ticket-popover-button-${id}`"
+                  href="#"
+                  :data-target="`#ticket-popover-${id}`"
+                  position="top-start">
+                  {{id}}
+                </a>
+                <section
+                  class="chi-popover"
+                  :id="`ticket-popover-${id}`"
+                  aria-modal="true"
+                  role="dialog"
+                  aria-label="Popover title"
+                >
+                  <header class="chi-popover__header">
+                    <h2 class="chi-popover__title">{{ id }}</h2>
+                  </header>
+                  <div class="chi-popover__content">
+                    <p class="chi-popover__text">Content for {{ id }}</p>
+                  </div>
+                </section>
+              </div>
+            </template>
+
+            <!-- Mounted -->
+            mounted() {
+              const buttonOpenOnHover = document.getElementById(`ticket-popover-button-${this.$props.id}`);
+              const popover = chi.popover(buttonOpenOnHover);
+              let hoverAnimationTimeout: number;
+
+              if (buttonOpenOnHover) {
+                buttonOpenOnHover.addEventListener('mouseenter', function() {
+                  hoverAnimationTimeout = setTimeout(() => {
+                    popover.show();
+                  }, 300);
+                });
+
+                buttonOpenOnHover.addEventListener('mouseleave', function() {
+                  if (hoverAnimationTimeout) {
+                    clearTimeout(hoverAnimationTimeout);
+                  }
+                  popover.hide();
+                });
+              }
+            }
+        #vertical-base-3.chi-tabs-panel(role='tabpanel')
+          :code(lang='html')
+            <!-- Template -->
+            <template>
+              <div class="chi-dropdown">
+                <button ref="dropdownTrigger" class="chi-button -icon -flat -bg--none -opacity-hover--80">
+                  <div class="chi-button__content">
+                    <i class="chi-icon icon-more-vert"></i>
+                  </div>
+                </button>
+                <div class="chi-dropdown__menu">
+                  <a class="chi-dropdown__menu-item" href="#">
+                    <span><i class="chi-icon icon-edit -sm -icon--primary" /></span>
+                    <span>Edit</span>
+                  </a>
+                  <a class="chi-dropdown__menu-item" href="#">
+                    <span><i class="chi-icon icon-delete -sm -icon--primary" /></span>
+                    <span>Delete</span>
+                  </a>
+                </div>
+              </div>
+            </template>
+
+            <!-- mounted -->
+            mounted() {
+              chi.dropdown(this.$refs.dropdownTrigger);
+            }
+  .chi-tabs-panel#html-portal-complex
+    :code(lang="html")
```

Comment:
@janibolkvadze I would suggest removing html blueprint as it doesn't match with the Vue code example